rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -366,14 +388,25 @@ namespace rocal{
                 )code"
             )
             .def(
-            "copy_data", [](rocalTensor &output_tensor, py::object p) {
+            "copy_data", [](rocalTensor &output_tensor, py::object p, RocalOutputMemType external_mem_type) {
                 auto ptr = ctypes_void_ptr(p);
-                output_tensor.copy_data(ptr);
+                output_tensor.copy_data(static_cast<void *>(ptr), external_mem_type);
             },
                 R"code(
                 Copies the ring buffer data to python buffer pointers.
                 )code"
             )
+            .def("copy_data_numpy", &copy_data_numpy_wrapper<u_char>, py::return_value_policy::reference)
+            .def("copy_data_numpy", &copy_data_numpy_wrapper<float>, py::return_value_policy::reference)
```

Comment:
Add a 
                R"code(
                Copies the ring buffer data to python buffer pointers.
                )code

                for these functions too